frontend/public/i18n.js

```diff
@@ -27,8 +28,8 @@ export const init = () => {
       backend: {
         loadPath: '/locales/resource.json?lng={{lng}}&ns={{ns}}',
       },
-      lng: localStorage.getItem('bridge/language'),
-      fallbackLng: 'en',
+      lng: getPreferredLanguage(),
+      fallbackLng: 'ja',
```

Comment:
Why is fallback `ja`?

Reply:
Ahh! I was testing.
forgot to clean that up.

Reply:
Ahh!
I was testing and overlooked while cleaning up.